vivo-project/Vitro
# Adding missing labels

**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues)**: [3886](https://github.com/vivo-project/VIVO/issues/3886)

# What does this pull request do?
Adding translations for some UI label keys

# What's new?
Translations for 19 UI label keys has been added in accordance with this file - [link](https://docs.google.com/spreadsheets/d/1goXRUsp5QlDGYu5sLOeggisjremBbN8v3iJ1ovFxQ3A/edit?usp=sharing). Thanks @litvinovg @michel-heon Jose Salm Anna Guillaumet for translations

# How should this be tested?

- Include support for languages in runtime.properties
- Run VIVO
- Change UI language/locale (to non en-US)
- Try to upload a file by following instructions described at https://github.com/vivo-project/Vitro/pull/251
- Check UI labels translations.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers
